src/t_hash.c

```diff
@@ -448,31 +445,31 @@ void hashTypeConvertListpack(robj *o, int enc) {
         /* Nothing to do... */
 
     } else if (enc == OBJ_ENCODING_HT) {
-        hashTypeIterator *hi;
+        hashTypeIterator hi;
         dict *dict;
         int ret;
 
-        hi = hashTypeInitIterator(o);
+        hashTypeInitIterator(o, &hi);
         dict = dictCreate(&hashDictType);
 
         /* Presize the dict to avoid rehashing */
         dictExpand(dict, hashTypeLength(o));
 
-        while (hashTypeNext(hi) != C_ERR) {
+        while (hashTypeNext(&hi) != C_ERR) {
             sds key, value;
 
-            key = hashTypeCurrentObjectNewSds(hi, OBJ_HASH_KEY);
-            value = hashTypeCurrentObjectNewSds(hi, OBJ_HASH_VALUE);
+            key = hashTypeCurrentObjectNewSds(&hi, OBJ_HASH_KEY);
+            value = hashTypeCurrentObjectNewSds(&hi, OBJ_HASH_VALUE);
             ret = dictAdd(dict, key, value);
             if (ret != DICT_OK) {
                 sdsfree(key);
                 sdsfree(value);              /* Needed for gcc ASAN */
```

Comment:
Clang-formats wants the comment to be aligned with the comment on the line below. (That's why the clang-format CI job failed.)

```suggestion
                sdsfree(value);               /* Needed for gcc ASAN */
```

Reply:
thanks for heads up. commited your change https://github.com/valkey-io/valkey/pull/1105/commits/a063effd54881b2ec1c1461aab9e934585d57384